docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,142 @@
+# Tizen Update Control Native API
+
+- [Privilege & Feature](#privilege-feature)
+- [Prerequisites](#prerequisites)
+- [Connecting to Server](#connecting-to-server)
+- [Checking the Latest Version](#checking-the-latest-version)
+- [Downloading and Updating to New Version](#downloading-and-updating-to-new-version)
+- [Making an Update Reservation](#making-an-update-reservation)
+- [Update Properties](#update-properties)
+
+You can control the system software version of IoT device using update control API.
+The control of system software version includes:
+- Checking the latest version
+- Downloading and updating to the latest version
+- Making a reservation for update
+
+The update procedure and the server may be different according to the platform developer.
+Therefore this API just offers general functions for update, and platform developers should develop the application using update control API properly.
+
+>As a default, the update agent works with [SmartThings Device Manager](https://console.smartthingsdm.com/) is available.
+
+## Privilege & Feature
+
+The update control API requires special privilege and feature to use it.
+
+| Privilege                                | Level    | Privacy      | Since | Description                              |
+|----------------------------------------|-------|------------|-----|----------------------------------------|
+| `http://tizen.org/privilege/updatecontrol.admin` | platform   | -      | 5.0   |  The application can control the firmware update procedure.       |
+
+| Feature key                              | Description                              | Since |
+|----------------------------------------|----------------------------------------|-----|
+| `http://tizen.org/feature/device_update`       | Specify this key, if the application requires Device Update API to control the system software update of the device. | 5.0   |
+
+More details about the privilege and feature, refer to [Security and API Privileges](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/security-and-api-privileges) and [Application Filtering](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/application-filtering).
+
+## Prerequisites
+
+To enable your application to use the update control functionality:
+
+1. To use the update control API at applications, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/updatecontrol.admin</privilege>
+   </privileges>
+```
+
+2. To use the functions and data types of the update control API, include the `<update_control.h>` header file in your application
+
+3. If some initialization is required to use update control functions, you can use `update_control_initialize()`
```

Comment:
Please add colon at the end of the sentence.

Reply:
Done.